src/porepy/grids/grid_bucket.py

```diff
@@ -940,11 +941,11 @@ def _find_shared_face(self, g0: pp.Grid, g1: pp.Grid, g_l: pp.Grid) -> np.ndarra
             cells 1 and 2 of g1 (second dimension of the array).
         """
         # Sort nodes according to node_number
-        g0, g1 = self.nodes_of_edge([g0, g1])
+        g0, g1 = self.nodes_of_edge((g0, g1))
```

Comment:
why this change?

Reply:
This is because `nodes_of_edge` expects a  `Tuple[pp.Grid, pp.Grid]`. 
However, this is of course a completely arbitrary choice, and you could equivalently extend the expected type of `nodes_of_edge` to include `List[pp.Grid, pp.Grid]`.
However, since the norm is that we represent edges as tuples, not lists, I made this change.

(on a practical level; I spotted this simply because my IDE, PyCharm, notified me of the inconsistency).

Reply:
Agreed, edges are tuples.